### PR TITLE
Pass along kwargs to backward

### LINKED ISSUE #103: Accelerator backward with retain graph
I want to use `loss.backward(retain_graph=True)` instead of normal backward. How can I do this using accelerate?

### PR DESCRIPTION
**Pull request by sgugger**
This PR allows the use of all keyword arguments for the `backward` method.

Fixes #103

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Great, LGTM!
